courses/machine_learning/deepdive/09_sequence_keras/translate/translate_model.ipynb

```diff
@@ -0,0 +1,809 @@
+{
```

Comment:
<ul><li>The list comprehension needs to be broken up (pep 8).</li><li>Why not using the built-in "open" here? Also, don't you need to close the stream when you are done reading from it (or use it in a context: "with etc.")?</li><li>maybe use .readlines() instead of read/strip/split</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/488/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>